servers/todo/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "perf-todo",
+  "version": "1.0.0",
+  "description": "Todo API reference app for Express performance benchmarking",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "dev": "nodemon server.js"
+  },
+  "keywords": [
+    "express",
+    "performance",
+    "benchmark",
+    "todo"
+  ],
+  "author": "jonchurch",
+  "license": "MIT",
+  "dependencies": {
+    "body-parser": "^2.2.0",
+    "compression": "^1.8.0",
+    "cookie-parser": "^1.4.7",
+    "cors": "^2.8.5",
+    "express": "^5.1.0",
+    "helmet": "^8.1.0",
```

Comment:
```suggestion
```

I am thinking that maybe we can just us our own libs to avoid external interference or at least anchor the version.

Reply:
FWIW I think there was a thread with the maintainer of `helmet` discussing bringing it into the org somewhere floating around. This one is so ubiquitous and also in our docs that I am comfortable with it ether way. That said, I think there is some value in splitting this apart somehow so we have more targeted tests as well.

Reply:
Imma keep it for now, my goal here is to represent something approaching a real world appplication while still having minimal business logic. I do expect folks are running helmet in prod, so this non business logic middleware fits the bill for me.